packages/vue/src/components/atoms/SfImage/SfImage.vue

```diff
@@ -4,12 +4,12 @@
     :style="imageStyle"
     data-testid="image-wrapper"
   >
-    <img
+    <component
+      :is="imageComponentTag"
       :loading="loading"
-      v-bind="$attrs"
+      v-bind="attributes"
       :src="src"
       :srcset="srcset"
-      :sizes="sizes"
```

Comment:
what is a reason for removing this?

Reply:
I removed it because it could collide with `sizes` prop from nuxt-img. I tested it once again and it overwrites prop from nuxt-img. Doing that I found more problems, cause `width` and `height` props also are dupllcated and also are required . So I need to make required property set to `false` for nuxt-img.

Reply:
I see but this prop is also needed in some cases for img tag

Reply:
Yes you are right, but I added it in `attributes` computed  property.